package.json

```diff
@@ -40,6 +40,7 @@
     "lodash": "^3.9.3",
     "ng-file-upload": "^12.2.13",
     "ng-image-gallery": "^2.1.4",
+    "npm": "^6.2.0",
```

Comment:
please remove. package.json is used by npm so it is a given that this is a dependency